src/tabs/default.scss

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import './tabs';
+
+@mixin iui-tabs-default {
+  @include iui-tabs;
+
+  > li {
+    > button {
+      height: $iui-baseline * 3;
```

Comment:
Shouldn't this use one of our height variables?

Reply:
I see the change, but now all the heights have changed drastically. Is this intentional?

Reply:
I changed all the tabs to use the input height and padding variables.  The borderless tab I multiplied the horizontal padding by 2 to get better visuals.